src/erf.jl

```diff
@@ -73,6 +76,17 @@ Compute the Dawson function (scaled imaginary error function) of `x`, defined by
 """
 dawson
 
+"""
+    faddeeva(z)
+
+Compute the Faddeeva function of complex `z`, defined by
+``e^{-z^2} \\operatorname{erfc}(-iz)``.
+Note that this function, also named `w` (original Faddeeva package) or `wofz` (Scilab package)
```

Comment:
missing a comma at the end of this line.

Reply:
yep, done